src/goto-symex/goto_symex_state.cpp

```diff
@@ -248,8 +248,8 @@ void goto_symex_statet::set_l1_indices(
     return;
   if(!ssa_expr.get_level_1().empty())
     return;
-  level0(ssa_expr, ns, source.thread_nr);
-  level1(ssa_expr);
+  ssa_exprt l0 = level0(std::move(ssa_expr), ns, source.thread_nr);
+  ssa_expr = level1(std::move(l0));
```

Comment:
Could we just do `ssa_expr = level1(level0(std::move(ssa_expr), ns, source.thread_nr));` please?